packages/manager/apps/web/client/app/hosting/email/detach/detach.routing.js

```diff
@@ -0,0 +1,68 @@
+import get from 'lodash/get';
+
+import component from './detach.component';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.hosting.detachEmail', {
+    url: '/detachEmail',
+    component: component.name,
+    resolve: {
+      goBack: /* @ngInject */ ($state) => () => $state.go('^'),
+      pricingType: /* @ngInject */ (
+        OVH_MANAGER_PRODUCT_OFFERS_PRICING_CONSTANTS,
+      ) =>
+        OVH_MANAGER_PRODUCT_OFFERS_PRICING_CONSTANTS.PRICING_CAPACITIES.DETACH,
+      workflow: /* @ngInject */ (
+        emailOptionDetachInformation,
+        emailOptionIds,
+        serviceName,
+        OVH_MANAGER_PRODUCT_OFFERS_WORKFLOW_CONSTANTS,
+        OVH_MANAGER_PRODUCT_OFFERS_PRICING_CONSTANTS,
+      ) => ({
+        options: {
+          ...emailOptionDetachInformation[0],
+          durationToUse: `${OVH_MANAGER_PRODUCT_OFFERS_PRICING_CONSTANTS.ISO_DURATION_FORMAT.PREFIX}12${OVH_MANAGER_PRODUCT_OFFERS_PRICING_CONSTANTS.ISO_DURATION_FORMAT.MONTH}`,
+        },
+        type:
+          OVH_MANAGER_PRODUCT_OFFERS_WORKFLOW_CONSTANTS.WORKFLOW_TYPES.SERVICES,
+      }),
+
+      onError: ($translate, Alerter) => (error) => {
+        Alerter.error(
+          $translate.instant('hosting_email_detach_option_error', {
+            error: get(error, 'data.message', error.message),
+          }),
+          'detach',
+        );
+      },
+      onSuccess: /* @ngInject */ ($translate, $window, Alerter, goBack) => (
+        detachResult,
+      ) => {
+        let successMessage;
+        if (!detachResult.autoPayWithPreferredPaymentMethod) {
+          $window.open(detachResult.url, '_blank');
```

Comment:
Is there a possibility that this could be blocked by the browser as a pop-up?

Reply:
Hmm good point, the tab opening will be blocked, the first time it occurs on the manager. I can make a workaround to bypass this behavior, but I prefer not to "hack" user's browser.
What do you think about ?

Reply:
Maybe you could display a message to say something like "If the link has not opened, click here to see it" or something like that instead

Reply:
@marie-j the message content has already a `<a>` tag to open the link

Reply:
I don't try to open the link personnaly, I don't want users to think the Manager is dosing unsafe behavior

Reply:
@FredericEspiau  I don't get your point

Reply:
Hmm after thinking about, maybe I should remove `$window.open`, as the popup might be blocked in many cases